modules/optimization/doc/content/source/materials/MisfitReporterOffsetFunctionMaterial.md

```diff
@@ -0,0 +1,47 @@
+# MisfitReporterOffsetFunctionMaterial
+
+!syntax description /Materials/MisfitReporterOffsetFunctionMaterial
+
+## Overview
+
+The `MisfitReporterOffsetFunctionMaterial` material is a specialized material that calculates a misfit value and its gradient with respect to a simulation variable, using a Gaussian function to weight the contribution of measurement points
+
+
+
+Measurement data:
+
+- Measurement values $\mathbf{u}_m = \{u_{m,1}, u_{m,2}, \ldots, u_{m,n}\}$
+- Measurement Locations: $\mathbf{p}_m = \{\mathbf{p}_{m,1}, \mathbf{p}_{m,2}, \ldots, \mathbf{p}_{m,n}\}$, where $\mathbf{p}_{m,i} = (x_{m,i}, y_{m,i}, z_{m,i})$ for 3D data
+
+- Simulation variable: $u(\mathbf{x})$
+- Gaussian beam width: $w$
+- Simulation point: $\mathbf{x}$
+
+The `misfit` at a given simulation point $\mathbf{x}$ is calculated as:
+
+$m(\mathbf{x}) = \sum_{i=1}^{n}  \left( u_{m,i} g(\mathbf{p}_{m,i}, \mathbf{x}) - u(\mathbf{x}) g(\mathbf{p}_{m,i}, \mathbf{x}) \right)^2,$
+
+where:
+
+- $g(\mathbf{p}_{m,i}, \mathbf{x})$ is the Gaussian weighting function, defined as:
+
+$g(\mathbf{p}_{m,i}, \mathbf{x}) = \exp\left( -\frac{2 \|\mathbf{p}_{m,i} - \mathbf{x}\|^2}{w^2} \right),$
+
+and the `misfit gradient` with respect to the simulation variable $u(\mathbf{x})$ is:
+
+$\frac{\partial m(\mathbf{x})}{\partial u(\mathbf{x})} = -2 \sum_{i=1}^{n} g(\mathbf{p}_{m,i}, \mathbf{x}) \left( u_{m,i} g(\mathbf{p}_{m,i}, \mathbf{x}) - u(\mathbf{x}) g(\mathbf{p}_{m,i}, \mathbf{x}) \right).$
```

Comment:
THis needs to be corrected for a general function and not just a gaussian.  You can keep the gaussian as an example but its not really relevant anymore.  Maybe reference the input file function:
!listing test/tests/optimizationreporter/function_misfit/forward_and_adjoint.i block=Functions